content/efcore/getting-started/samples/defining-entity-relationships/one-to-one-manual-foreign-key/README.md

```diff
@@ -0,0 +1,9 @@
+# One-to-One Relationships with Manual Foreign Key
+
+This code example demonstrates how to create a one-to-one relationship by manually setting the foreign key using Fluent API.
+
+## Code Files:
+- [Program.cs](../shared/Program.cs)
+- [Reader.cs](../shared/Reader.cs)
+- [Address.cs](../shared/Address.cs)
+- [LibraryContext.cs](LibraryContext.cs)
```

Comment:
Make `LibraryContext.cs` the second item in the list so that it is one of the first files to appear in the console.